### PR TITLE
Chrome: Use an expandable panel for the block's advanced settings

### LINKED ISSUE #2851: Group additional class name and anchor under a panel in block inspector
In https://github.com/WordPress/gutenberg/pull/2820 we added support for panels within the block inspector.

We should use this to group the generic "advanced" settings for blocks, currently "additional class name" and "anchor" id.

### PR DESCRIPTION
**Pull request by youknowriad**
## Description
closes #2851

## Screenshots (jpeg or gifs if applicable):
screen shot 2017-10-02 at 14 09 10

### CODE REVIEW

**Comment by mtias** [comment]
@jasmussen should we call this one advanced? Looping back to our early mockups here :)

**Comment by jasmussen** [comment]
Yes, I think that makes sense, otherwise it's "Block" all over.

**Comment by youknowriad** [comment]
@jasmussen so just "Advanced" ? or more like "Advanced Settings"?

**Comment by jasmussen** [comment]
I'm okay with either "Advanced" or "Advanced Settings". I like Advanced as it's a bit shorter, and ideally we'll only put stuff in here that really you'd only want to customize once in a blue moon.

File: editor/sidebar/block-inspector/style.scss [hunk 1]
```diff
@@ -34,6 +34,10 @@
 	}
 }
 
+.editor-block-inspector__content .editor-advanced-controls {
+	margin-bottom: -14px;
```
Comment (gziolo) [inline editor/sidebar/block-inspector/style.scss]: Why not `-16px` like in other places? There is still a small whitespace added with the current value:

screen shot 2017-10-03 at 11 58 06

**Review by gziolo — APPROVED** [review]
I think it can land as soon as text is updated and margin-bottom is confirmed. It works as expected.

File: editor/sidebar/block-inspector/style.scss (hunk 1 above)
Comment (youknowriad) [reply editor/sidebar/block-inspector/style.scss]: Because of the focus style when clicked.

Comment (gziolo) [reply editor/sidebar/block-inspector/style.scss]: @jasmussen should have a better idea what's best. I'm only asking to learn myself :)

**Pull request merged**